### PR TITLE
Misc test stubs types

### PR DESCRIPTION
**Pull request by UnknownPlatypus**
Some more misc test stubs update

### CODE REVIEW

File: django-stubs/test/html.pyi [hunk 1]
```diff
@@ -1,12 +1,15 @@
 from collections.abc import Sequence
 from html.parser import HTMLParser
+from re import Pattern
 from typing import Any
 
 from typing_extensions import TypeAlias
 
-WHITESPACE: Any
+ASCII_WHITESPACE: Pattern
```
Comment (sobolevn) [inline django-stubs/test/html.pyi]:
```suggestion
ASCII_WHITESPACE: Pattern[str]
```

**Review by sobolevn — COMMENTED** [review]
Thanks!

File: django-stubs/test/html.pyi (hunk 1 above)
Comment (UnknownPlatypus) [reply django-stubs/test/html.pyi]: Thanks, I've updated other `Pattern` which I believe also miss this.

While at it, I also updated the `allowlist.txt` to reference every regex pattern in the `Lazy regex` section. 
I found some missing running `rg ": Pattern" -N | cut -d":" -f2 | xargs -I {} rg {} ./scripts/stubtest/allowlist_todo.txt`

**Review by sobolevn — APPROVED** [review]
Thanks!

**Pull request merged**